randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -888,8 +929,14 @@ Extra - room_id: [26]
                       Movement (Beginner) and Can Use Bombs
               # Crumble jumps
               Movement (Expert)
-          # Damage Runs
-          Varia Suit
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              All of the following:
+                  # No springball: https://youtu.be/a4urCWRPTNM
+                  Damage Runs (Hypermode) and Cold Damage ≥ 733
+                  Mid-Air Morph (Beginner) or Can Use Bombs
```

Comment:
this is both mid air morph and bombs, no?
Also wonder whether it should really be hypermode. conceptually its not *that* much different from doing it with spring.

Reply:
> this is both mid air morph and bombs, no?

oops i didnt think about that i guess ill just remove mid air morph then since that since you need bombs to do this.

> Also wonder whether it should really be hypermode. conceptually its not that much different from doing it with spring.

bombs take significantly longer to detonate than to use springball, and in that time difference is a knockback from the cold damage, which makes it worth the separatation imo

Reply:
yeah I'd likely agree with timing bombs between cold hits makes it harder rather than the instantaneous springball (not that I have/will try it)